interactive-bots/helloworld/dub125/notes.md

```diff
@@ -0,0 +1 @@
+Only Issue I had was that I was try to run "zulip-run-bot helloworld --config-file /Users/dylanwichman/Downloads/zuliprc "  on my server... but I quickly found out that I needed to be doing it on my desktop!"
```

Comment:
Please line wrap at 79 characters. This is done so that `git diff` is readable in terminals.